### DISCUSSION TITLE
How to add customized IP Address role?

### DISCUSSION THREAD

**Discussion by nikl15** [netbox-community/netbox#6286] [Q&A] [answered]
Hi! I'm absolutely new by using Netbox. Can you tell me how can i add my own IP address role? And can i add Customize field as another column to table?

**Comment by candlerb** [comment, marked as answer]
> Can you tell me how can i add my own IP address role?

You can't - it is a fixed set.  If you have a use case that you think is important, you can raise it as a feature request on github to have a new role added.  However, you can now apply tags and custom fields to IP addresses, so you should consider if you can meet your needs using those features first.

> And can i add Customize field as another column to table?

Not quite sure what you mean.  What version of Netbox are you running?

You can add [custom fields](https://netbox.readthedocs.io/en/stable/additional-features/custom-fields/) to most data models in Netbox, and recent versions of Netbox let you customize the columns displayed in tables, including custom fields.  Click on the (cog) "Configure" button when viewing a table of search results, and you'll get a pop-up letting you select which columns to view - including individual custom fields.

**Reply by nikl15** [reply]
Thanks for your answer! very helpful!
